.circleci/config.yml

```diff
@@ -272,7 +287,6 @@ jobs:
       # repeats some of the server tests, because these fail fast and short-curcuit us before the long-running browser tests
       - run: (cd server && npm run unit_tests)
       
-      - run: npm run check_format client
```

Comment:
Note/Reasoning: for whatever reason, this doesn't doesn't actually throw an error